examples/https-gke/variables.tf

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "name" {
+  default = "tf-lb-https-gke"
+}
+
+variable "service_port" {
+  default = "30000"
+}
+
+variable "service_port_name" {
+  default = "http"
+}
+
+variable "target_tags" {
+  default = "tf-lb-https-gke"
+}
+
+variable "backend" {}
+
+data "google_client_config" "current" {}
```

Comment:
This should be in `main.tf` of the example.

Reply:
Thank you for review, @aaron-lane! Didn't notice that as it's in the middle of vars, previously existed. 
[Fixed](https://github.com/terraform-google-modules/terraform-google-lb-http/compare/f8191af405d3015066dd99ee72dea8f79138b83f..60a3e4fd0058e6a4b03c77b054ecd78ce6aa341e)